releng/org.eclipse.glsp.repository/test

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
```

Comment:
The file name looks strange. Was this a test and is a left-over or is this required and could be named differently?

Reply:
Good catch. Yes this is a leftover and can be remvoed